From 908df6ddfa9dcb9bb8093c91d3759d1bdbec19b2 Mon Sep 17 00:00:00 2001
From: sp1rit <sp1rit@disroot.org>
Date: Sun, 23 May 2021 18:55:26 +0200
Subject: [PATCH] tweaks to llvm clang build

Signed-off-by: sp1rit <sp1rit@disroot.org>
---
 main.cpp    | 4 ++--
 meson.build | 3 +--
 2 files changed, 3 insertions(+), 4 deletions(-)

diff --git a/main.cpp b/main.cpp
index 61d25dd..aada02f 100644
--- a/main.cpp
+++ b/main.cpp
@@ -9,8 +9,8 @@ int main (int argc, char *argv[])
 	
 	Glib::RefPtr<Gtk::Application> app = Gtk::Application::create(argc, argv, "com.github.blackhole89.notekit");
 	
-	app->signal_activate().connect( [app,&mainwindow]() {
-			mainwindow=new CMainWindow(app);
+	app->signal_activate().connect( [app]() {
+			auto mainwindow=new CMainWindow(app);
 			app->add_window(*mainwindow);
 		} );
 	
diff --git a/meson.build b/meson.build
index 34bba29..a9553f3 100644
--- a/meson.build
+++ b/meson.build
@@ -1,6 +1,6 @@
 project('notekit', 'cpp',
 	version : '0.1.0',
-	default_options : ['warning_level=3', 'cpp_std=c++14']
+	default_options : ['warning_level=3', 'cpp_std=c++17']
 )
 
 conf_data = configuration_data()
@@ -52,4 +52,3 @@ executable('notekit',
 	install : true
 )
 
-meson.add_install_script('build-aux/postinstall.py')
-- 
2.31.1

